Dockerfile

```diff
@@ -0,0 +1,46 @@
+# для deploy
+FROM python:3.8
+
+WORKDIR /divorce_project
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update \
+    && apt-get install netcat -y \
+    && apt-get install -y dos2unix
+RUN apt-get upgrade -y && apt-get install postgresql gcc python3-dev musl-dev -y
+
+RUN pip install --upgrade pip
+COPY ./requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN dos2unix ./entrypoint.sh && apt-get --purge remove -y dos2unix && rm -rf /var/lib/apt/lists/*
```

Comment:
There's no sense in removing apt/lists/* and dos2unix here as it's anyway saved in the previous layers and you will not save space in Docker image